docs/source/datasets/index.rst

```diff
@@ -42,80 +42,123 @@ The following simulant attributes are included in this dataset:
    :header-rows: 1
 
    * - Attribute Name
-     - Column Name    
+     - Column Name
+     - Notes    
    * - Unique simulant ID
      - :code:`simulant_id`
+     - Not affected by noise functions; intended use is 'ground truth' for PRL tracking. 
    * - First name
      - :code:`first_name`
+     - 
    * - Middle initial
      - :code:`middle_initial`
+     - 
    * - Last name
      - :code:`last_name`
+     - 
    * - Age
-     - :code:`age`  
+     - :code:`age` 
+     - Rounded down to an integer. 
    * - Date of birth
      - :code:`date_of_birth`
+     - Formatted as MM/DD/YYYY.
    * - Physical address street number
      - :code:`street_number`
+     - 
    * - Physical address street name
      - :code:`street_name`
+     - 
    * - Physical address unit
      - :code:`unit_number`
+     - 
    * - Physical address city
-     - :code:`city`    
+     - :code:`city` 
+     -    
    * - Physical address state
      - :code:`state`  
+     - 
    * - Physical address ZIP code
      - :code:`zipcode`
+     - 
    * - Relationship to person 1 (head of household)
      - :code:`relationship_to_household_head` 
-   * - Sex (binary; 'male' or 'female')
+     - 'Person 1', 'head of household', and 'Reference person' are all synonymous in this context. Possible values for this indicator include:
+       Reference person; Biological child; Adopted child; Stepchild; Sibling; Parent; Grandchild; Parent-in-law; Child-in-law; Other relative;
+       Roommate; Foster child; and Other nonrelative.
+   * - Sex 
      - :code:`sex`  
+     - Binary; 'male' or 'female'.
    * - Race/ethnicity
      - :code:`race_ethnicity` 
+     - The following exhaustive and mutually exclusive categories for the single composite 'race/ethnicity' indicator are as follows:
+       White; Black; Latino; American Indian and Alaskan Native (AIAN); Asian; Native Hawaiian and Other Pacific Islander (NHOPI); and
+       Multiracial or Some Other Race. 
 
 Household Surveys: ACS and CPS
 ------------------------------
 There are two simulated household survey datasets that can be used: the American
 Communities Survey (ACS) and the Current Population Survey (CPS). 
 
+
 .. list-table:: **Simulant attributes**
    :header-rows: 1
 
    * - Attribute Name
      - Column Name
+     - Notes
    * - Unique simulant ID
-     - 
+     - simulant_id
+     - Not affected by noise functions; intended use is 'ground truth' for PRL tracking. 
    * - Household ID 
-     -  
+     - :code:`household_id` 
+     - 
```

Comment:
```suggestion
     - Not affected by noise functions; intended use is 'ground truth' for PRL tracking.
```
The current `household_id` column is also the "ground truth" and is not intended for linking or noising.